src/main.cpp

```diff
@@ -254,8 +250,10 @@ int main(int argc, char **argv)
   // Load parameters from private node handle.
   std::string port;
   int32_t baud;
-  ros::param::param<std::string>("~port", port, "/dev/ttyUSB0");
-  ros::param::param<int32_t>("~baud", baud, 115200);
+
+  ros::NodeHandle imu_nh("imu"), private_nh("~");
```

Comment:
Yes. Parameters are in private namespace, topics are all published on the 'imu' namespace as per convention.